fala/settings/base.py

```diff
@@ -128,55 +130,26 @@
             '()': 'logstash_formatter.LogstashFormatter'
         }
     },
-    'filters': {
-        'require_debug_false': {
-            '()': 'django.utils.log.RequireDebugFalse'
-        },
-        'require_debug_true': {
-            '()': 'django.utils.log.RequireDebugTrue'
-        }
-    },
     'handlers': {
-        'mail_admins': {
```

Comment:
:neckbeard: To be pedantic, removing `mail_admins` log handler is not strictly part of the change in this commit. If someone wants to look _why_ it was removed in this commit, they have to go to the pull request and then to the JIRA ticket. It may be nice to mention the causes in the commit message or create a separate commit.